tests/data/serialization/ndjson/test_classification.py

```diff
@@ -4,13 +4,18 @@
 from labelbox.data.serialization.ndjson.converter import NDJsonConverter
 from labelbox.data.serialization.ndjson.objects import NDLine
 
+IGNORE_KEYS = ['unit', 'page']
```

Comment:
Instead of popping these keys off of the serialized result each time, can we remove these keys from the output of `serialize` if the value is None, at least for page and unit? I feel that it'd be confusing to users if they started seeing these fields set to None for any other type of data. 

Maybe play around with `exclude_none` setting for the pydantic `model.dict()` function?

Reply:
Updated the `serialize` logic to pop if the value is `None`. Opted for this instead of setting `exclude_none` because it would affect some other fields that we may not necessarily want to exclude (one example is that it would pop off feature schema in some scenarios). If we want to opt for using this field in the future, it will likely require some additional effort